config/config.go

```diff
@@ -610,9 +610,11 @@ type ScrapeConfig struct {
 	// More than this label value length post metric-relabeling will cause the
 	// scrape to fail. 0 means no limit.
 	LabelValueLengthLimit uint `yaml:"label_value_length_limit,omitempty"`
-	// More than this many buckets in a native histogram will cause the scrape to
-	// fail.
+	// More than this many buckets in a native histogram will automatically merge buckets
+	// to stay within the limit.
```

Comment:
Let's wordsmith this into a somewhat smoother English. How about: "If there are more than this many buckets in a native histogram, buckets will be merged to stay within the limit."

Reply:
Agreed